README.md

```diff
@@ -1,6 +1,51 @@
 # spellcheck
 
-This repository contains a Docker image with the [R `spelling` package](https://cran.r-project.org/web/packages/spelling/index.html) installed.
-The role of this repository is to facilitate spell checking actions across `AlexsLemonade` repositories.
+This repository contains a GitHub action to run the [R `spelling` package](https://cran.r-project.org/web/packages/spelling/index.html).
+The role of this action is to facilitate spell checking actions across `AlexsLemonade` repositories.
 
-speeling
+Currently the action will only spell check text in Rmd and md files
+
+## Usage
+
+To use this action, create a `.github/workflows/spellcheck.yml` file in your repository with the following contents (modified to fit your exact needs):
+
+```yaml
+name: Check Spelling
+on:
+  pull_request:
+    branches:
+     - main
+
+jobs:
+  spell-check:
+    runs-on: ubuntu-latest
+    name: Spell check files
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Spell check action
+        uses: alexslemonade/spellcheck
+        id: spell
+        with:
+          dictionary: components/dictionary.txt
+
+      - name: Upload spell check errors
+        uses: actions/upload-artifact@v2
```

Comment:
If we bump this to v4 (and add some `write` permissions), we can have nice things:

In the context of OpenScPCA-analysis, we'd post the artifact link to an issue. 
In the context of other repos where we do have a PR trigger, we'd post as a PR comment.

Reply:
https://github.com/sjspielman/test-spellcheck/blob/25ba084de3ccf7977f6fb40e70a3647aebdd70a5/.github/workflows/spellcheck.yml#L31-L38

Reply:
updated